app/models/pieces/pawn.rb

```diff
@@ -11,18 +11,105 @@ def valid_move?(destination_row, destination_column)
     end
   end
 
+  # Idea is that once a pawn moves up/down one row, it's no longer eligible to be captured
+  # through en passant. I also added another field to the pieces table so that the game
+  # can keep track the pawn's en passant status. Not sure if this is the best way to
+  # track en passant status
+  def change_enpassant_status
+    case color
+    when 'black'
+      update_attributes(en_passant: false) if row_coordinate == 5
+    when 'white'
+      update_attributes(en_passant: false) if row_coordinate == 2
+    end
```

Comment:
I forget, what happens if the pawn moves up 2 spaces in the first move? It should change to `false` the too, right?